package.json

```diff
@@ -2,8 +2,8 @@
   "name": "@puppeteer/replay",
   "version": "0.2.0",
```

Comment:
the title says version 1.0.0?

Reply:
Temporary; couldn't think of a better name